convmelspec/stft.py

```diff
@@ -211,13 +202,7 @@ def forward_on_the_fly(
         hop_size: int = 512,
         power: bool = True,
         window: torch.Tensor = None,
-        coreml: bool = False,
     ):
-
-        # Only needed for Core ML
-        if coreml:
```

Comment:
@nikalra Does on-the-fly mode still work for CoreML?

Reply:
It does, but this is now built into coremltools via the `pass_pipeline` option so we don't need to hack it in anymore.

https://apple.github.io/coremltools/source/coremltools.converters.convert.html#module-coremltools.converters._converters_entry:~:text=pass_pipelinePassPipeline